src/charts/index.js

```diff
@@ -1,8 +1,10 @@
-import legend from './legend/legendComponent.js';
-import tooltip from './tooltip/tooltipComponent.js';
-import stackedArea from './stackedArea/stackedAreaComponent.js';
+import donut from './donut/DonutComponent';
```

Comment:
Thanks for fixing the 'capitalization' of these, we have it also in an upcoming PR